### ISSUE TITLE
Wrong behavior when rating Hard and Good on new cards

### ISSUE THREAD

**Issue by Andy-Rossi** [hugomarins/fsrs4remnoteFork#1]
With the default settings, when rating a new card as "good", the new due date is 1 day.

**But** if I rate a new card as "Partially recalled" first and 15 minutes later rate it as "good", then the **due date become 4 days!** I think this is not intended, as it doesn't make sense. It should be 1 day.

**Comment by NNGTHB** [comment]
same

**Comment by hugomarins** [comment]
@WandersonRossi and @NNGTHB, this is according to the formula expressed in https://github.com/hugomarins/fsrs4remnoteFork#initial-stability.

The sense is that, in your next rating (15m if you set so), the next stability will be less than if you have rated good (in this case, in your next rating - 1 day after - your first stability would be 7 days). The idea is that you first rating is important in evaluating the hardness of that content to you. I understand if looks strange in the first glance, but you have to remember you rated the card an additional time 15 min later (this rating corresponds to the one made next day if you have rated Good instead of Hard).

**Comment by Andy-Rossi** [comment]
I still think it is weird, since this doesn't take in consideration the learning process. A new card may be "hard" because it has new information to the user, not because it is actually hard to remember. 

And if it is actually hard, that's more reason for the interval after getting it right to be shorter (1 day) and not longer (4 days)!

I might be a biased based on how anki's scheduler work with it's initial fixed learning steps. But if you say that is intended behavior based on the original creator's formula, than I guess there is nothing that can be done in that :/ I just avoid using "hard" on new cards, and use forgot instead in case it was "hard".

(don't get me wrong though, I love the plugin's scheduling otherwise! I just don't like that specific thing).

**Comment by hugomarins** [comment]
@WandersonRossi I got your point, but the interval is already being shorter (instead of 7 days, 4 days). You must reckon the additional learning step 15 m later (otherwise you will be comparing things that cannot be compared: the _first_ interval after graduating the card first rated Good [after completing the learning step default to one day] with the _second_ interval after graduating the card first rated Hard [also after completing the learning step, now default to 10 min]).

You can change these default learning steps to whatever you feel more comfortable with (reducing the learning step difference between pressing Hard and Good will make this impression of failure in the formula fade away):
![image](https://github.com/hugomarins/fsrs4remnoteFork/assets/54500828/127341ed-21fd-4d90-8859-2c55c168aabd)

The formula makes sense if you consider this fact.  Look at the formula and you will see that it is impossible to have a greater value for Hard than for Good, if you compare them in the same moment (first stability interval, reached only after completing the learning step of an additional rating): $S_0(G) = w_0 + (G-1) \cdot w_1$

Yet, I think that in practice it will not be relevant, as the next interval will make things clearer. These intervals are always arbitrary anyway, and in my experience it will not have any negative effect. Later stabilities (intervals) will be more critical.

Last, but not least, I don't know how to code; I just adjusted and corrected behaviors that bothered me too much. I don't know how to change this [you would need to insert an additional learning step in case of pressing Hard, just like Anki does]. You have to keep in mind that in Anki, pressing Hard repeats the learning step in which you are. In FSRS4RemNote, there is always only one step. I can't replicate Anki's behavior, and I also don't think it would be that important.
